### ISSUE TITLE
Button to "Jump to main content" stopped working

### ISSUE THREAD

**Issue by JoaoGabrielD** [LemmyNet/lemmy-ui#2714]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

The feature implemented and demonstrated on #1158 has since stopped working. As described in that pull request:

> Adds ability to press the "Tab" key to reveal a hidden "Jump to conent" link. Clicking or hitting enter will bring the post listings regardless of page into focus.

Note that since I don't use keyboard navigation **I can't say how recent this regression is**.

### Steps to Reproduce

1. After page load, press the tab key to select the hidden button
2. Press the enter key to "click" it

### Technical Details

Error thrown in console:
![image](https://github.com/user-attachments/assets/df1493b4-464d-417f-a0af-74722f5877cc)

### Lemmy Instance Version

0.19.3

### Lemmy Instance URL

https://programming.dev/

**Comment by dessalines** [comment]
Can you try replicating this on https://voyager.lemmy.ml ? It worked there, and it might've been a bug that's already been fixed.

**Comment by JoaoGabrielD** [comment]
> Can you try replicating this on https://voyager.lemmy.ml ? It worked there, and it might've been a bug that's already been fixed.

Still the same.
![image](https://github.com/user-attachments/assets/acf4bf26-a3ff-423c-a22c-cb6f0647f30b)

Didn't mention in the post, but this happens on both Firefox and Chrome, so I don't think this is platform specific.

I'm not a FE developer, but I noticed a couple of things on `src/shared/components/app/app.tsx`.
I reckon this stopped working on this commit:
https://github.com/LemmyNet/lemmy-ui/pull/2419/files#diff-2a025b2ede840b33110c1865cd85576f9f438d1f5e577bddef0a21abcf4bc91b

`handleJumpToContent()` lost a line and the onClick event in the button was also changed
